mlflow/models/utils.py

```diff
@@ -203,17 +203,6 @@ def _coerce_to_pandas_df(input_ex):
             elif np.isscalar(input_ex):
                 input_ex = pd.DataFrame([input_ex])
             elif not isinstance(input_ex, pd.DataFrame):
-                try:
-                    import pyspark.sql.dataframe
-
-                    if isinstance(input_example, pyspark.sql.dataframe.DataFrame):
-                        raise MlflowException(
-                            "Examples can not be provided as Spark Dataframe. "
-                            "Please make sure your example is of a small size and "
-                            "turn it into a pandas DataFrame."
-                        )
```

Comment:
Do we have a test for this exception? Looks like we don't.

Reply:
Not a blocker, can be addressed in a follow-up.

Reply:
We can tell user to use `spark_dataframe.toPandas()` to convert it to pandas dataframe in the error message.

Reply:
Makes sense, let me add that quickly in this PR:)